extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -145,64 +119,64 @@ export const SignTransaction = () => {
-      <El>
-        <HeaderEl>Confirm Transaction</HeaderEl>
-        {flaggedKeyValues.length ? (
-          <FlaggedWarningMessage
-            isUnsafe={isUnsafe}
-            isMalicious={isMalicious}
-            isMemoRequired={isMemoRequired}
-          />
-        ) : null}
-        {!isDomainListedAllowed && !isSubmitDisabled ? (
-          <FirstTimeWarningMessage />
-        ) : null}
-        <PunycodedDomain domain={domain} />
-        <SubheaderEl>
-          This website is requesting a signature on the following{" "}
-          {isFeeBump ? "fee bump " : ""}transaction:
-        </SubheaderEl>
-        {isFeeBump ? (
-          <>
-            <TransactionHeader
-              _fee={_fee}
-              _sequence={_sequence}
-              source={_feeSource}
-              isFeeBump
-              isMemoRequired={isMemoRequired}
-            />
-            <InnerTransactionWrapper>
-              <Transaction
-                flaggedKeys={flaggedKeys}
-                isMemoRequired={isMemoRequired}
-                transaction={_innerTransaction}
-              />
-            </InnerTransactionWrapper>
-          </>
-        ) : (
+    <ModalWrapper>
+      <ModalInfo
+        domain={domain}
+        domainTitle={domainTitle}
+        subject={`This website is requesting a signature to the following${" "}
+            ${isFeeBump ? "fee bump " : ""}transaction:`}
+        title="Confirm Transaction"
+      >
+        <TransactionHeader
+          _fee={_fee}
+          _sequence={_sequence}
+          source={source}
+          isFeeBump={isFeeBump}
+          isMemoRequired={isMemoRequired}
+        />
+      </ModalInfo>
+      {flaggedKeyValues.length ? (
+        <FlaggedWarningMessage
+          isUnsafe={isUnsafe}
+          isMalicious={isMalicious}
+          isMemoRequired={isMemoRequired}
+        />
+      ) : null}
+      {!isDomainListedAllowed && !isSubmitDisabled ? (
+        <FirstTimeWarningMessage />
+      ) : null}
+
+      {isFeeBump ? (
+        <InnerTransactionWrapper>
           <Transaction
             flaggedKeys={flaggedKeys}
             isMemoRequired={isMemoRequired}
-            transaction={transaction}
+            transaction={_innerTransaction}
           />
-        )}
-
-        <ButtonContainer>
-          <RejectButtonEl size="small" onClick={() => rejectAndClose()}>
-            Reject
-          </RejectButtonEl>
-          <SubmitButton
-            isValid={!isSubmitDisabled}
-            isSubmitting={isConfirming}
-            size="small"
-            onClick={() => signAndClose()}
-          >
-            Confirm Transaction
-          </SubmitButton>
-        </ButtonContainer>
-      </El>
-    </>
+        </InnerTransactionWrapper>
+      ) : (
+        <Transaction
+          flaggedKeys={flaggedKeys}
+          isMemoRequired={isMemoRequired}
+          transaction={transaction}
+        />
+      )}
+
+      <div className="SignTransaction--button-container">
```

Comment:
5) Here's an example of me using `Button` components from SDS. I needed them to be in a horizontal row, so I created a custom element to wrap around them to get them where I needed to be instead of applying any styles directly to the `Button` itself